src/Employer/Employer.Web/Views/VacancyPreview/AdvertPreview.cshtml

```diff
@@ -10,22 +10,32 @@
   <a asp-route="@RouteNames.EmployerCheckYourAnswersGet" asp-all-route-data="@Model.RouteDictionary" esfa-automation="link-back" class="govuk-back-link">Back to check your answers</a>
 }
 
-<h1 class="govuk-heading-xl">Preview your advert</h1>
+<div class="govuk-grid-column-two-thirds-from-desktop">
```

Comment:
This needs to be in its own row as otherwise its going to appear in the advert preview

Reply:
Amended in latest.